sdk/src/main/java/com/bugsnag/android/FileStore.java

```diff
@@ -69,7 +72,8 @@ String write(@NonNull T streamable) {
 
         Writer out = null;
         try {
-            out = new FileWriter(filename);
+            FileOutputStream fos = new FileOutputStream(filename);
+            out = new BufferedWriter(new OutputStreamWriter(fos, "UTF-8"));
```

Comment:
do any of these objects also need closing?

Reply:
Addressed by other inline comment on JsonStream